CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 - The DNS configuration can now be passed to the pod via the command line ([#2040](https://github.com/coreos/rkt/pull/2040)). See [`DNS support`](https://github.com/coreos/rkt/blob/master/Documentation/networking.md#dns-support) documentation.
 - Errors are now structured, allowing for better control of the output. See [#1937](https://github.com/coreos/rkt/pull/1937) for background and the [Error & Output](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md#errors--output) section of the Hacking document for usage.
 - All output now uses the new log package in `pkg/log` to provide a more clean and consistent output format and more helpful debug output. See [#1937](https://github.com/coreos/rkt/pull/1937)
+- Added configuration for stage1 image. User can drop a configuration file to `/etc/rkt/stage1.d` (or to `stage1.d` in user configuration directory) to tell rkt to use different name, version and location of stage1 image instead of build-time defaults ([#1977](https://github.com/coreos/rkt/pull/1977)).
+- Replaced `--stage1-image` flag with a new set of flags. `--stage1-url`, `--stage-path`, `--stage1-name` do the usual fetching from remote if image does not exist in store. `--stage1-hash` takes stage1 image directly from store. `--stage1-from-dir` works together with the default stage1 images directory and is described in the next point ([#1977](https://github.com/coreos/rkt/pull/1977)).
```

Comment:
the `--stage1-image` flag...

"if the image does not exist in the store"

"takes the stage1 image directly from the store"